src/Admin/Admin.js

```diff
@@ -21,13 +30,45 @@ const Admin = ({ setAdminSelections }) => {
       console.log("Admin form is not complete")
     } else if (selectedAction === "Add new menu item") {
       console.log("Add new")
-      setAdminSelections({ selectedRestaurant: selectedRestaurant, selectedAction: selectedAction })
+      setAdminSelections(
+        {
+          selectedRestaurant: selectedRestaurant,
+          selectedAction: selectedAction,
+          restaurantId: restaurantId()
+        })
       navigate("/add-menu-item")
     } else {
       console.log("Delete existing")
-      setAdminSelections({ selectedRestaurant: selectedRestaurant, selectedAction: selectedAction })
+      setAdminSelections(
+        {
+          selectedRestaurant: selectedRestaurant,
+          selectedAction: selectedAction,
+          restaurantId: restaurantId()
+        })
       navigate("/admin/delete")
     }
+    console.log("ADMIN ADMIN SELECGTIONS", { selectedRestaurant: selectedRestaurant, selectedAction: selectedAction, restaurantId: restaurantId() })
+  }
+
+  const restaurantFieldAlert = () => {
+    if (
+      selectedRestaurant === "None selected" || !selectedRestaurant
+    ) {
+      return (
+        <span style={{ color: "rgba(255, 0, 0, 0.9)", fontSize: "5vh" }}>*</span>
```

Comment:
Maybe when we refactor we can move this styling to the CSS file to keep everything consistent?